scss/sveetoy/components/_buttons.scss

```diff
@@ -70,7 +70,7 @@ $sveetoy-button-colors-schemes: () !default;
     div, p, dd, li{
         button, .button{
             &:last-child{
-                margin-bottom: 0;
```

Comment:
`sveetoy` is a saas library built on top of Foundation, and the code has been copied to the repo. So we shouldn't modify it directly, you should update _off.scss instead.
Besides, you add a margin after every button in a div/p/li/dd here, which is not what we want (we want to fix only the described issue)